schemas/io-package.json

```diff
@@ -20,14 +20,30 @@
         "type"
       ],
       "properties": {
```

Comment:
Why was ``_id``removed etc? It is required to be of type string and it is needed at the instanceObjects..

Reply:
And why having just the link properties there? Or do I miss anything?

Reply:
it was Merge error. Now fixed